docs/features/software-templates/writing-custom-field-extensions.md

```diff
@@ -32,7 +32,12 @@ As an example, we will create a component that validates whether a string is in
 import React from 'react';
 import { FieldExtensionComponentProps } from '@backstage/plugin-scaffolder-react';
 import type { FieldValidation } from '@rjsf/utils';
-import FormControl from '@material-ui/core/FormControl';
+import {
+  FormControl,
+  FormHelperText,
+  Input,
+  InputLabel,
+} from '@material-ui/core';
```

Comment:
Sorry I realized that top level imports are not allowed. 
So import needs to be in the form of `import FormControl from '@material-ui/core/FormControl';` `import InputLabel from '@material-ui/core/InputLabel';` and so on.

Reply:
Don't worry! Adjustment done!